logfire/_internal/main.py

```diff
@@ -1261,6 +1278,10 @@ def instrument_httpx(
             capture_request_form_data: Set to `True` to capture the request form data.
                 Specifically captures the `data` argument of `httpx` methods like `post` and `put`.
                 Doesn't inspect or parse the raw request body.
+            request_hook: A function called right after a span is created for a request.
+            response_hook: A function called right before a span is finished for the response.
+            async_request_hook: A function called right after a span is created for an async request.
+            async_response_hook: A function called right before a span is finished for an async response.
```

Comment:
Please apply anything you'd like here.

Reply:
I do think we should improve this but it'll be easier to do that along with other docs, so let's merge this PR because it's easily an improvement but keep this whole docstring in mind.